tests/x11/rstudio_desktop.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Basic test of RStudio Desktop
+# Maintainer: Dan Čermák <dcermak@suse.com>
+
+use base "x11test";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use rstudio;
+
+sub run {
+    mouse_hide(1);
+    ensure_installed('rstudio-desktop');
+
+    x11_start_program('rstudio', target_match => 'rstudio_desktop-main-window');
+    wait_still_screen;
+
+    rstudio_help_menu(rstudio_mode => "desktop");
+
+    rstudio_sin_x_plot(rstudio_mode => "desktop");
+
+    rstudio_create_and_test_new_project(rstudio_mode => "desktop");
+
+    # bye-bye
+    wait_screen_change { send_key('alt-f4') };
```

Comment:
You should not need a `wait_screen_change` as for all "x11" tests the post_run_hook ensures to wait for "generic-desktop". You overwrite the post_run_hook but then I suggest to still call the base class one additionally. E.g. after line 38 add `$self->SUPER::post_run_hook;`. As an alternative replace the `wait_screen_change` with an `assert_screen 'generic-desktop';`

Reply:
I have opted to adding `$self->SUPER::post_run_hook;`